modules/elements-color-picker/assets/js/editor/commands/show-swatches.js

```diff
@@ -0,0 +1,157 @@
+import CommandBase from 'elementor-api/modules/command-base';
+import Utils from '../../../../../../core/app/assets/js/utils/utils';
+
+export class ShowSwatches extends CommandBase {
+	constructor( args ) {
+		super( args );
+
+		this.colors = {};
+		this.pickerClass = 'elementor-element-color-picker';
+		this.pickerSelector = '.' + this.pickerClass;
+		this.container = null;
+		this.tmpImages = [];
+
+		this.COLORS_LIMIT = 5;
```

Comment:
Maybe as a const on top of the file?

Reply:
Yup, you're right. don't know why I didn't do it from the beginning...